### DISCUSSION TITLE
Bitbucket auth with `enableTeamGroups` only works for Workspace Admins

### DISCUSSION THREAD

**Discussion by ringods** [concourse/concourse#6526] [General]
Hello,

I'm just posting this here to inform any other possible Concourse user using the Bitbucket auth connector with the `enableTeamGroups` setting. This only works for workspace admins. The call used in `dex` to retrieve the groups is not accessible by a normal user.

Here is my bugreport at the upstream `dex` project which includes part of my communication with Atlassian Support:
https://github.com/dexidp/dex/issues/1977

Hope this prevents other people loosing time on this.

Ringo

**Comment by piroux** [comment]
Thanks a lot.

I should have posted here too when I tried a few months ago, right before and right after Atlassian enforced [breaking changes of their Bitbucket workspace/users API](https://github.com/dexidp/dex/issues/1811), as I had no successful results too.

Quoting you from the dex issue:
> On our end, we will not go forward with the Bitbucket Cloud connector anymore.

Same here!

---

:rainbow: If anyone in the future is able to setup a Concourse RBAC with Bitbucket groups through the Bitbucket Cloud connector of dex, please ping us and please share your setup :pray:
